### ISSUE TITLE
Show only latest order count

### ISSUE THREAD

**Issue by mazharoddin** [owen2345/camaleon-ecommerce#38]
Hi @owen2345,
I feel its better to show only latest order count, instead of showing all orders count.
Currently all orders count is shown here, instead of that showing the count for new orders will be helpful to know new orders quickly.
<img width="163" alt="screen shot 2016-08-27 at 12 29 54 pm" src="https://cloud.githubusercontent.com/assets/6102465/18025794/229e1bf0-6c52-11e6-842a-9ab607273c53.png">

**Comment by owen2345** [comment]
maybe count orders with specific status? 
what about "paid" status?

**Comment by mazharoddin** [comment]
yeah, we can have different filters for different status like paid, shipped etc, but here showing only new orders will help users to get know how many new orders came at given time.
Otherwise user need to click on orders and go inside to check if there are any new orders.
What do you feel..

**Comment by p-decoraid** [comment]
:+1: for showing new orders

**Comment by mazharoddin** [comment]
Thanks.
